.github/workflows/preflight.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       matrix:
         vm_size: [""]
         parallelism: [20]
-        index: [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19]
+        index: [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]
```

Comment:
just making sure, but why are we adding another index here? wouldn't that be OOB?

Reply:
Ah, oops, for some reason I thought it was an index per test, not runner